packages/2019-disaster-game/src/components/Game/KitScreen/index.js

```diff
@@ -15,10 +15,10 @@ import {
 } from "../../../state/chapters";
 import {
   getKitCreationItems,
-  addItemToPlayerKit,
-  getPlayerKit
+  getPlayerKit,
+  addItemToPlayerKit
 } from "../../../state/kit";
-import { addPoints, addBadge } from "../../../state/user";
+import { earnBadge as _earnBadge, getBadges } from "../../../state/tasks";
```

Comment:
There's a mix of syntax used for actionCreators are referred to in this file and it's a bit confusing. Some have the _actionCreator prefix, and some don't. Should probably be consistent.

For the future, interested in what the idiomatic / most common way is.

Reply:
Updated all action creators to follow _actionCreator syntax